composable/assetlist.json

```diff
@@ -229,6 +229,359 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/tinkernet/images/tnkr.svg"
       }
+    },
+    {
+      "description": "The native token of Ethereum, bridged via IBC.",
+      "denom_units": [
+        {
+          "denom": "eth-wei",
+          "exponent": 0
+        },
+        {
+          "denom": "eth",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/F9D075D4079FC56A9C49B601E54A45292C319D8B0E8CC0F8439041130AA7166C",
+      "name": "Ethereum",
+      "display": "eth",
+      "symbol": "ETH",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "eth-wei",
```

Comment:
On ethereum, the base denom is just "wei"

Reply:
we never ended up changing this, can I add it as an alias to the eth folder?

Reply:
it must refer to the "base" property, not just any alias on that denom, which is why this must be "wei"